query_sheet.rb

```diff
@@ -10,6 +10,7 @@
 MASTER_SCHEMA_SPREADSHEET_ID = '1KABFR083wl6Ok0WEIsPs1lefZt7U9PJz1iuneQ7Prc0'
 NB_EXPORT_SPREADSHEET_ID = ARGV[1]
 INVALID_ADDRESSES_SPREADSHEET_ID = '1dlC9ZM1tMLW5XazyR6BVRDRapbllx_d6gko--41p7a8'
+NULL_ADDRESSES_SPREADSHEET_ID = '[NULL_ADDRESSES_SPREADSHEET_ID]'
```

Comment:
this needs to be updated with the spreadsheet ID of the null addresses spreadsheet, whenever it is created